hack/gen_statik.sh

```diff
@@ -19,22 +19,23 @@ set -euo pipefail
 export GOFLAGS="-mod=vendor"
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 BIN=${DIR}/bin
-LICENSES=${BIN}/licenses
 STATIK=${BIN}/statik
 
 mkdir -p ${BIN}
 
-if ! [[ -f ${LICENSES} ]]; then
-  pushd ${DIR}/tools
-  echo >&2 'Installing licenses tool'
-  GOBIN=${BIN} GO111MODULE=on go install -tags tools github.com/google/trillian/scripts/licenses
-  popd
-fi
-
 TMP_DIR=$(mktemp -d)
 trap "rm -rf $TMP_DIR" EXIT
 
-${LICENSES} save "github.com/GoogleContainerTools/skaffold/cmd/skaffold" --save_path="${TMP_DIR}/skaffold-credits"
+# Copy licenses
+pushd vendor
+LICENSES=$(find . \( -type f -name 'LICENSE*' -or -name 'COPYING*' -or -name 'NOTICE*' \))
```

Comment:
Integrating with licenses is important and was intentional, so that we don't have to maintain this logic. 
This line does not cover the logic of licenses. For example there is a requirement that Mozilla Public License licenses need to include the source as well. 

The package moved to https://github.com/google/go-licenses and we will have to switch over to that. But I'd rather use that then this logic. This gives us flexibility and maintainability on the long run as well, e.g. if we switch over to downloading go modules instead of vendor directory, the library will work just as well, while this line will break.

Reply:
OK. Makes sense. I tried https://github.com/google/go-license and it's slow as hell now. We need to find a way to make it faster or to make it not run each time we `make install`

Reply:
What do you mean slow as hell?